app/styles/_variables.scss

```diff
@@ -41,3 +41,8 @@ $color-shadow-gray-light: #ccc;
 $color-shadow-gray-dark: #898787;
 
 $color-text-placeholder-grey-dark: #666;
+
+$brand-success: #357935;
+$brand-info: #1b6d85;
```

Comment:
What brand is this? Is it overriding values from osf-style? bootstrap?

At some point we (or an eventual designer) need to go through all our colors and come up with a new palette that we actually use/follow...

Reply:
This is a [bootstrap-via-osf-style](https://github.com/CenterForOpenScience/osf-style/blob/2e811db48d076f73ba2cb5cf40e8e7ffb8726270/bower_components/bootstrap/less/variables.less#L17) thing. 

I think once we go to Bootstrap 4 there's a [better mechanism](http://getbootstrap.com/docs/4.0/getting-started/theming/#modify-map) for this

Reply:
Ah, okay. We could switch to using maps like that without bootstrap 4, but it will be nicer once there's a central place to override.